lib/util/content/file.content-builder.js

```diff
@@ -0,0 +1,11 @@
+const is = require('ramda/src/is')
+const CommonBuilder = require('./common.content-builder')
+
+class FileContentBuilder extends CommonBuilder {
+
+  constructor(url, type) {
+    super(url, type)
```

Comment:
tohle je spatne ne? `payload` ma byt objekt a ne jenom ta url

```
"content": {
    "type": "image",
    "payload": {
      "url": "https://upload.wikimedia.org/wikipedia/commons/1/11/V%C3%A1clav_Havel_cut_out.jpg"
    }
  }
```

Reply:
To teda!!! 👍